File: Programming-languages-and-laboratory/Java/Exercises/09_compito_java_2016_06_15/Giocatore.java
public class Giocatore {
    protected int codiceSquadra;
    protected String cognome;
    protected int eta;
    protected int numeroMaglia;
    protected String ruolo;
    protected Boolean titolare;

    public Giocatore(int codiceSquadra, String cognome, int eta, int numeroMaglia, String ruolo, Boolean titolare) {
        this.codiceSquadra = codiceSquadra;
        this.cognome = cognome;
        this.eta = eta;
        this.numeroMaglia = numeroMaglia;
        this.ruolo = ruolo;
        this.titolare = titolare;
    }
}

File: Programming-languages-and-laboratory/Python/Exercises/30_compito_java_20190617/Giocatore.py
class Giocatore:
    def __init__(self, codice_squadra, cognome, nome, titolare, eta, numero_maglia, ruolo):
        self._codice_squadra = codice_squadra
        self._cognome = cognome
        self._nome = nome
        self._titolare = titolare
        self._eta = eta
        self._numero_maglia = numero_maglia
        self._ruolo = ruolo

    def get_codice_squadra(self):
        return self._codice_squadra

    def __str__(self):
        return self._nome + "\t" + self._cognome + "\t" + str(self._eta) + "\t" + str(self._numero_maglia) + "\t" + self._ruolo
